EU4toV2/Source/Mappers/Buildings/Buildings.cpp

```diff
@@ -41,12 +29,14 @@ void mappers::Buildings::registerKeys()
 	registerRegex("[a-zA-Z0-9_]+", [this](const std::string& buildingName, std::istream& theStream) {
 		Building building(theStream);
 		buildings.insert(std::make_pair(buildingName, building));
-		});
-	registerRegex("[a-zA-Z0-9_\\.:]+", commonItems::ignoreItem);
+	});
+	registerRegex(commonItems::catchallRegex, commonItems::ignoreItem);
 }
 
 std::optional<mappers::Building> mappers::Buildings::getBuilding(const std::string& buildingName) const
 {
-	if (buildings.count(buildingName)) return std::make_optional(buildings.at(buildingName));
-	return std::nullopt;
-}
\ No newline at end of file
+	if (buildings.contains(buildingName))
+		return buildings.at(buildingName);
```

Comment:
If you don't want to do lookup twice:
```suggestion
	if (const auto& building = buildings.find(buildingName); building != buildings.end())
		return building->second();
```